recipes/tblite-python/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "tblite-python" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tblite/tblite/releases/download/v{{ version }}/tblite-{{ version }}-source.tar.xz
+  sha256: 8c554bc0dec6770ffdd708bc7db21cb009f1ab602cc8d1a04be0d9107be33177
+
+build:
+  number: 0
+  skip: true  # [osx and py<38]
+  skip: true  # [win]
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - meson
+    - ninja
+    - python
+    - pkg-config
+  host:
+    - cffi
+    - pip
+    - python
+    - tblite >={{ version }}
+  run:
+    - cffi
+    - numpy
+    - python
+
+
+test:
+  imports:
+    - tblite
+    - tblite.library
+    - tblite.interface
+  requires:
+    - pip
+    - pytest
+    - setuptools
+    - ase
+  commands:
+    - pip check
+    - pytest --pyargs tblite -vv  # [build_platform == target_platform]
```

```suggestion
    - pytest --pyargs tblite -vv
```
I _believe_ that the `# [build_platform == target_platform]` is unnecessary here, because tests aren't run for cross-compiles, unless running emulated?

---

Sometimes I run tests in the cross-compiling environment, because of the consistency checks like `pip check` or `pkg-config {{ name }} --exact-version {{ version }}`. But I can drop this selector until I add the OSX/Arm64 migrator later.